frontend/internal-packages/agent/src/chat/workflow/nodes/designSchemaNode.ts

```diff
@@ -137,15 +39,13 @@ export async function designSchemaNode(
   if (!createVersionResult.success) {
     const errorMessage =
       createVersionResult.error || 'Failed to create new version'
-    await logAssistantMessage(state, repositories, 'Version creation failed')
+    await logAssistantMessage(state, repositories, errorMessage)
```

Comment:
In fact, if saving to DB fails, it is a 500 status error and should be retried as is.
I am planning to fix it next in the form of using LangChain's Command.